FWCore/ParameterSet/test/test_catch2_ps.cc

```diff
@@ -0,0 +1,875 @@
+/*
+ */
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include <algorithm>
+#include <iostream>
+#include <limits>
+#include <string>
+#include <cassert>
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/types.h"
+#include "FWCore/Utilities/interface/EDMException.h"
+#include "FWCore/Utilities/interface/Algorithms.h"
+#include "FWCore/Utilities/interface/Digest.h"
+namespace edm {
+  bool operator!=(const edm::EventRange& iLHS, const edm::EventRange& iRHS) {
```

Comment:
Would this operator overload be useful in general?

Reply:
Unclear. I'm pretty sure it was in the original cppunit version.